src/wasm/literal.cpp

```diff
@@ -56,13 +73,11 @@ Literal::Literal(const Literal& other) : type(other.type) {
       new (&exn) std::unique_ptr<ExceptionPackage>();
     }
   } else if (other.isGCData()) {
-    // Avoid calling the destructor on an uninitialized value
-    new (&gcData) std::shared_ptr<Literals>(other.gcData);
+    new (&gcData) std::shared_ptr<GCData>(other.gcData);
   } else if (type.isFunction()) {
     func = other.func;
   } else if (type.isRtt()) {
-    // Nothing to do: Rtts help JITs optimize, but are not used in the
-    // interpreter yet, and they are opaque to the wasm itself.
+    new (&rttSupers) auto(std::make_unique<RttSupers>(*other.rttSupers));
```

Comment:
```suggestion
    new (&rttSupers) std::unique_ptr<RttSupers>(*other.rttSupers);
```

Reply:
This is trying to make a `unique_ptr` by giving the constructor a value of the type. But it expects a pointer of the type (which it would then own). So this doesn't compile.

I think we have to call `make_unique` here, or call `new` and pass that to the `unique_ptr` constructor, as we are allocating new data here.

Reply:
Yes, makes sense. This is analogous to the case of non-empty exception package above.